@xen-orchestra/web/src/types/xo/host.type.ts

```diff
@@ -36,9 +40,18 @@ export type XoHost = {
     cores: number
     sockets: number
   }
+  logging: {
+    syslog_destination?: string
+  }
+  CPUs: {
+    modelname: string
```

Comment:
CPUs type is: `Record<string string>`
Therefore, it is not guaranteed that the CPUs properties are present.
If you want to use autocompletion for a property, do the following:
```suggestion
    modelname?: string,
    [key: string]: string | undefined
```